src/task_processor/processor.py

```diff
@@ -194,7 +194,7 @@ def _run_task(
         "result": result.lower(),
     }
 
-    timer.labels(**labels)  # type: ignore[no-untyped-call]
+    timer.labels(**labels)
```

Comment:
I don't really know why mypy pulled me up on this in this PR given that I've not touched this file, or any related code...